app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/AccountNumber.scala

```diff
@@ -16,10 +16,13 @@
 
 package uk.gov.hmrc.cdsreimbursementclaim.models.claim
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.Format
+import uk.gov.hmrc.cdsreimbursementclaim.utils.SimpleStringFormat
 
 final case class AccountNumber(value: String) extends AnyVal
 
 object AccountNumber {
-  implicit val format: OFormat[AccountNumber] = Json.format[AccountNumber]
+
+  implicit val accountNumberFormat: Format[AccountNumber] =
+    SimpleStringFormat(AccountNumber(_), _.value)
```

Comment:
The account number same account name extends AnyVal to avoid unnecessary memory allocation, but generated 
```
Json.format[AccountNumber]
```
forced to create object anyway to store it.
This is I just serialise string field back and forward.